### DISCUSSION TITLE
Home Assistant non più accessibile con Fritz 7590

### DISCUSSION THREAD

**Discussion by jinx978** [indomus/forum#2471] [HUB: Home Assistant]
Ciao a tutti
avendo cambiato da poco il vecchio modem (avevo prima il modem Wind) con un Fritz 7590, pensavo di migliorare la situazione e la gestione con HA , ma purtroppo non è così.
Non riesco più ad accedere ad HA tramite il dominio, fino a qualche giorno fa attivo, di DuckDns.
Esiste una guida che mi spieghi le operazioni da fare per far colloquiare Fritz e HA?
Il modem riconosce HA, lo vedo nell'elenco delle connessioni, ma non mi fa accedere alla pagina "miodominio.ducksdns.org".

Grazie mille in anticipo a chi mi aiuterà

**Comment by snakuzzo** [comment]
Premesso che è completamente errato pensare al Fritzbox come il risolutore di tutti i mali, le casistiche sono tante.
Vai sul sito e fai una ricerca con la parola "remoto". Troverai tutti gli articoli che ti servono

**Comment by labionda67** [comment]
Ciao, domanda: hai aperto la porta 8123 sul fritz indirizzata allo HA?

**Comment by vince84be** [comment]
mi unisco alla conversazione.
oggi ho cambiato provider adsl e sono passato ad aruba con un fritzbox. purtroppo non riconosce il raspberry con homeassistant. ho anche resettato tutto e ho fatto un installazione pulita di ha ma all'avvio non riesco ad accedere al sito homeassistant.local:8123
